components/AuthButton.tsx

```diff
@@ -1,9 +1,11 @@
+'use client'
+
 import Link from 'next/link';
-import { cookies } from 'next/headers';
```

Comment:
Not sure why this file was touched. It doesn't pertain towards the task.

Reply:
`pnpm build` failed because `cookies` in line 1 and `cookieStore` in line 6 is not being used.